src/FormHandler/ThreadFormHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+
+namespace BiffBangPow\MessageBoard\FormHandler;
+
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use BiffBangPow\MessageBoard\Model\Thread;
+
+class ThreadFormHandler
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    private $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+
+        $this->entityManager = $entityManager;
+    }
+
+    public function handle(Request $request){
+        $thread = new Thread();
+
+        $thread -> setTitle((htmlspecialchars($request->get('title'))));
+        $thread -> setContent((htmlspecialchars($request->get('content'))));
```

Comment:
you don't need to put spaces either side of arrow. These sorts of comments might seem a bit pedantic but there is a coding standard for PHP PSR-2 http://www.php-fig.org/psr/psr-2/